mirrord/layer/src/file/filter/read_local_by_default.rs

```diff
@@ -13,6 +13,7 @@ pub fn regex_set_builder() -> RegexSetBuilder {
         r"^.+\.pyc$",
         r"^.+\.py$",
         r"^.+\.jar$",
+        r"^.+\.class$",
```

Comment:
This and all similar regexes could be simplified to `r".\.class$"`, but doesn't have to be in this PR.